libscript/include/pivot/script/Stack.hxx

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "pivot/ecs/Core/Data/value.hxx"
+
+#include <functional>
+#include <unordered_map>
+
+namespace pivot::ecs::script::interpreter
+{
+
+struct Variable {
+    std::string name;
+    bool hasValue = false;
+    data::Value value;
+    std::unordered_map<std::string, Variable> members;
+};
```

Comment:
Is considered by gcc as a forward declaration, no idea why
@Ersikan

Reply:
So we understood that it's because of `std::unordered_map<std::string, Variable>` which is a pretty nooby mistake by me.
Since you may only store pointers or references, I'm guessing I'll be using `std::unique_ptr<Variable>`.
This should be fine, right ?

Reply:
After the review from @Ersikan , I just removed it since it was a useless re-implementation of pivot::ecs::data::Value.